scripts/install/deploy/install_redis.sh

```diff
@@ -1,8 +1,26 @@
 #!/bin/sh
 
-# @todo #240:30min Implement redis availability check and install flows. In case
-#  Redis is not available lets install it and launch.
-#  In case it is installed but not among current process it should be launched only.
-#  Please also check if all the dependency checks are in place besides redis and implement rest if any.
+# Script for Redis availability check, installation and launch
 
-echo "Implement redis availability check"
+which redis-cli
+if [ "$?" -gt "0" ]; then
+  echo "Redis Not installed, installing"
+  sudo apt install make gcc libc6-dev tcl
+  wget http://download.redis.io/redis-stable.tar.gz
+  tar xvzf redis-stable.tar.gz
+  cd redis-stable
+  sudo make install
+  echo "Done installing Redis"
+else
+  echo "Redis already installed"
+fi
+
+echo "Redis PING"
+redis-cli ping
+if [ "$?" -gt "0" ]; then
+  echo "Redis Not running, launching"
+  src/redis-server > /dev/null &
```

Comment:
@vladarefiev not sure, I'd add this as different puzzle to create a service file and start the service in case it is needed. @emilianodellacasa do we need to have a service for the redis-server so it will be started after restart of a host where the deployment is happening to?

Reply:
@ekondrashev It would be better, thanks

Reply:
@ekondrashev The same should be true for Postgres, after a restart of the server all should be up and running